composer.json

```diff
@@ -11,17 +11,17 @@
     "require": {
         "php": "^7.1",
         "nikic/php-parser": "^3.1",
-        "symfony/console": "^3.2",
-        "symfony/yaml": "^3.2",
+        "symfony/console": "^3.2|^4",
```

Comment:
The correct syntax is `||` not `|` (https://getcomposer.org/doc/articles/versions.md#version-range)